README.md

```diff
@@ -88,23 +88,23 @@ running churn
++-----------------------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+
+| file                                                | stink_score                | churn                      | cost                       | coverage                   |
++-----------------------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+
+| lib/skunk/cli/commands/default.rb                   | 166.44                     | 3                          | 0.5548                     | 0                          |
+| lib/skunk/cli/application.rb                        | 139.2                      | 3                          | 0.46399999999999997        | 0                          |
+| lib/skunk/cli/command_factory.rb                    | 97.6                       | 2                          | 0.488                      | 0                          |
+| test/test_helper.rb                                 | 75.2                       | 2                          | 0.376                      | 0                          |
+| lib/skunk/rubycritic/analysed_module.rb             | 48.12                      | 2                          | 0.8592                     | 72.72727272727273          |
+| test/lib/skunk/cli/commands/status_reporter_test.rb | 45.6                       | 1                          | 0.456                      | 0                          |
+| lib/skunk/cli/commands/base.rb                      | 29.52                      | 3                          | 0.0984                     | 0                          |
+| lib/skunk/cli/commands/status_reporter.rb           | 8.0                        | 3                          | 2.6652                     | 100.0                      |
+| test/lib/skunk/rubycritic/analysed_module_test.rb   | 2.63                       | 2                          | 1.3156                     | 100.0                      |
+| lib/skunk.rb                                        | 0.0                        | 2                          | 0.0                        | 0                          |
+| lib/skunk/cli/options.rb                            | 0.0                        | 2                          | 0.0                        | 0                          |
+| lib/skunk/version.rb                                | 0.0                        | 2                          | 0.0                        | 0                          |
+| lib/skunk/cli/commands/help.rb                      | 0.0                        | 2                          | 0.0                        | 0                          |
++-----------------------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+
```

Comment:
@bronzdoc We still display the `churn_times_cost` column. So these changes are no longer needed.

Reply:
This was included in this PR by mistake